### PR TITLE
Fix image resize filter from object

### PR DESCRIPTION
**Pull request by asturur**
Note on changes on image initialize:

We are calling initElement, that calls setElement, that is calling initConfig and applyFilters.
So the part after this is just duplicated.

### CODE REVIEW

**Comment by asturur** [comment]
couple of test needed

File: src/shapes/image.class.js [hunk 1]
```diff
@@ -88,18 +88,10 @@
...
       this.filters = [ ];
       this.resizeFilters = [ ];
       this.callSuper('initialize', options);
-
       this._initElement(element, options);
-      this._initConfig(options);
-
-      if (options.filters) {
-        this.filters = options.filters;
-        this.applyFilters();
```
Comment (kangax) [inline src/shapes/image.class.js]: How would the filters be applied now during image initialization?

Comment (kangax) [inline src/shapes/image.class.js]: Oh, I see your explanation. Sorry, got it!

**Comment by kangax** [comment]
After tests, I'm ready to merge then!

**Comment by asturur** [comment]
@kangax any help on why the test is failing? it gives no clue. The row is:
```
ok(filterFromObj instanceof fabric.Image.filters.Resize);
```

**Comment by asturur** [comment]
I commented the part of the test that i added and does not run under node.

**Comment by asturur** [comment]
@kangax it is finally done.

**Comment by kangax** [comment]
+Changelog

**Pull request merged**